db/migrate/20200611102258_add_unique_reference_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueReferenceToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :unique_user_reference, :string, null: false, unique: true
```

Comment:
The schema file doesn't have the unique index. Might have to `add_index` separately.

Reply:
oh yeah, didn't spot that